packages/client/hmi-client/src/components/workflow/ops/interventions/tera-interventions-operation.ts

```diff
@@ -0,0 +1,27 @@
+import { InterventionPolicy } from '@/types/Types';
+import { WorkflowOperationTypes } from '@/types/workflow';
+import type { Operation, BaseState } from '@/types/workflow';
+
+export interface InterventionsState extends BaseState {
+	transientInterventionPolicy: InterventionPolicy;
```

Comment:
Why is this labelled as transient?
Is this not just the State's `InterventionPolicy`?

Reply:
true we dont really change the state's intervention policy, Ive renamed to `interventionPolicy`